config/locales/sufia.en.yml

```diff
@@ -166,3 +168,5 @@ en:
       department:
         text: "Department"
         label_html: "Department"
+      open:
+         label_html: Open Access
```

Comment:
let me check into this. Open Access shows up other places, so I'm not sure what's going on.